src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -63,27 +123,28 @@ const ReworkRequest = () => {
             }}
             icon={<NavigateBackIcon className="orange" />}
             size="large"
-            disabled={!currentStage?.previous}
+            disabled={disableReworkButton || !currentStage?.previous}
           />
         </Tooltip>
       </DialogTrigger>
       <DialogSurface>
         <DialogBody>
           <DialogTitle>Rework request</DialogTitle>
           <DialogContent>
-            <p>
-              Send request back a step?
-              <br />
-              Fill out this modal with a form for sending a request back a step
-              with a reason. (Notes capability pending)
-            </p>
+            <Field label="Rework reason" required>
+              <Textarea value={reason} onChange={updateReason} required />
+            </Field>
           </DialogContent>
           <DialogActions>
             <DialogTrigger disableButtonEnhancement>
               <Button appearance="secondary">Cancel</Button>
             </DialogTrigger>
             <DialogTrigger disableButtonEnhancement>
-              <Button appearance="primary" onClick={updateHandler}>
+              <Button
+                disabled={!updateReason}
```

Comment:
I think you meant this....
```suggestion
                disabled={!reason}
```